pex/pex.py

```diff
@@ -626,6 +640,27 @@ def execute_interpreter(self):
             code.interact()
             return None
 
+    def execute_with_options(self, options, args):
+        """
+        Restart the process passing the given options to the python interpreter
+        """
+        # Find the pex bootstrap location
+        # We expect this environment variable to be set by __main__.py
+        main = sys.modules.get("__main__")
+        if main:
+            run_pex = os.path.dirname(main.__file__)
+            interpreter = PythonInterpreter.get().binary
+            cmdline = [interpreter] + options + [run_pex] + args
+            TRACER.log(
+                "Re-executing with interpreter options: "
+                "cmdline={cmdline!r}, ".format(
+                    cmdline=" ".join(cmdline),
+                )
+            )
+            os.execv(interpreter, cmdline)
+        else:
+            return "Unable to resolve pex path"
```

Comment:
This should be impossible; so its not very important except for the code reader, but it would be spot on if it said "Unable to resolve installed pex path".